graylog2-web-interface/src/stores/streams/StreamRulesStore.ts

```diff
@@ -75,6 +76,9 @@ class StreamRulesStore {
   _emitChange() {
     this.callbacks.forEach((callback) => callback());
   }
+  unregister(callback: Callback) {
+    lodash.pull(this.callbacks, callback);
```

Comment:
Then please introduce the explicit dependency.

To be honest I rather stick to the standard library whenever possible, specially when there is an equivalent for the utility functions introduced in other libraries like lodash or jQuery, but we don't have a rule for that.

In my opinion the addition of lodash to the toolchain is bearable for the current team, but we will face challenges (and to be fair not just caused by lodash) when new team members arrive. The amount of technologies to learn in order to write some frontend code is really huge. We are not alone at this, as many node and JS projects have the same issue, but I think without a careful selection of the libraries we use, the problem will be even worse.